Library/C510_R1FA035_Orange.lib.vkp

```diff
@@ -1,8 +1,8 @@
 ;C510 SW-R1FA035 Orange
-;���������� �������
+;���������� �������

```

Comment:
Please rebase. I've updated most of Cyrillic comments in elfpack/library/consts to avoid codepage fails like this one